products/create-products.js

```diff
@@ -0,0 +1,31 @@
+function createdProduct(camera) {
```

Comment:
You only need this or the createCameraLi function, not both. They kinda do the same thing, creating the dom for one camera object